docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,107 @@
+# Help FastAPI - Get Help
+
+Você gosta do **FastAPI**?
+
+Gostaria de ajudar o FastAPI, outros usuários, e o autor?
+
+Ou gostaria de obter ajuda para utilizar o **FastAPI**?
+
+Há maneiras muito simples de ajudar (algumas envolvem somente um ou dois cliques). 
+
+Há várias maneiras de buscar ajuda também.
+
+## Star **FastAPI** no Github
```

Comment:
```suggestion
## Estrele **FastAPI** no Github
```

Reply:
Acredito que Favorite se encaixa melhor no contexto. @cassiobotaro o que vc acha?

Reply:
Pessoal acredito que Star e Watch sejam termos como proprios do GitHub. Similares al Pull Request. 
Por exemplo aqui na documentacao em pt https://help.github.com/pt/github/getting-started-with-github/be-social
Note que o termo Watch esta sendo traduzido como Watch em negrito. Acredito, que o mesmo se aplique a o termo Star e Fork

Reply:
Sim, concordo